apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html

```diff
@@ -71,6 +72,9 @@
 
                 <!-- Section to create New Sales Invoice, Cash Invoice, Credit Note and Debit Note -->
                 <div class="btn-group desktop-view-btn" dropdown *ngIf="selectedVoucher === 'sales'" [autoClose]="true">
+                    <div class="export-butn-wrap">
```

Comment:
Please check the spelling `butn` if it can be `btn` or `button` as used throughout the app